packages/types/src/strategies.ts

```diff
@@ -13,6 +13,7 @@ export type BackupCodeStrategy = 'backup_code';
 export type ResetPasswordPhoneCodeStrategy = 'reset_password_phone_code';
 export type ResetPasswordEmailCodeStrategy = 'reset_password_email_code';
 export type CustomOAuthStrategy = `oauth_custom_${string}`;
+export type EnterpriseSSOStrategy = 'enterprise_sso';
```

Comment:
🔧 We should mark `SamlStrategy` as deprecated in favor of EnterpriseSSOStrategy:
```ts
/**
 * @deprecated Use `EnterpriseSSOStrategy` instead
 */
export type SamlStrategy = 'saml';
```

Reply:
Fixed [f9c2194](https://github.com/clerk/javascript/pull/4596/commits/f9c2194678113a36fdcbb84bb55042d170b9e230)

Thanks for catching this, we need to mention it in the API docs as well I'll update them!

Reply:
Well reminded on the docs, we'll need to update it here:

https://github.com/clerk/clerk-docs/blob/3a68d1a415b9928f69b01afbb8936bcc719a877c/docs/custom-flows/saml-connections.mdx?plain=1#L47